iceoryx2-bb/elementary/src/math.rs

```diff
@@ -102,22 +102,23 @@ impl<const N: usize> ToB64 for [u8; N] {
             if 16 < N - i {
                 let mut data = [0u8; 16];
                 data.copy_from_slice(&self[i..i + 16]);
-                result.push_str(&unsafe { core::mem::transmute::<[u8; 16], u128>(data) }.to_b64());
+                result.push_str(&u128::from_ne_bytes(data).to_b64());
```

Comment:
While clippy is right and the transmute is equivalent to `from_ne_bytes`, I think this was broken on big endian machines. I'm pretty sure this needs to be `from_le_bytes` in order to make this work on big endian.